internal/eth/txn_test.go

```diff
@@ -1985,6 +1986,44 @@ func TestCallAndProcessReply(t *testing.T) {
 
 }
 
+func TestCallAndProcessReplyPrivateCall(t *testing.T) {
+	qm := &messages.QueryTransaction{
+		SendTransaction: messages.SendTransaction{
+			Method: &ethbinding.ABIElementMarshaling{
+				Name: "method1",
+				Outputs: []ethbinding.ABIArgumentMarshaling{
+					{Name: "arg1", Type: "uint256"},
+				},
+			},
+			TransactionCommon: messages.TransactionCommon{
+				PrivacyGroupID: "HvuYiHoyL31rZ5OYRzSYMGGRV+V8KS4LMuHRfmfDWzg=",
+			},
+		},
+	}
+
+	rpc := &testRPCClient{
+		resultWrangler: func(retString interface{}) {
+			retVal := "0x000000000000000000000000000000000000000000000000000000000003039"
+			reflect.ValueOf(retString).Elem().Set(reflect.ValueOf(retVal))
+		},
+	}
+
+	tx, err := NewSendTxn(&qm.SendTransaction, nil)
+	assert.NoError(t, err)
+	assert.Equal(t, "HvuYiHoyL31rZ5OYRzSYMGGRV+V8KS4LMuHRfmfDWzg=", tx.PrivacyGroupID)
+
+	res, err := tx.CallAndProcessReply(context.Background(), rpc, "latest")
+	assert.NoError(t, err)
+	assert.Equal(t, "priv_call", rpc.capturedMethod)
+	assert.Equal(t, "HvuYiHoyL31rZ5OYRzSYMGGRV+V8KS4LMuHRfmfDWzg=", rpc.capturedArgs[0])
+	assert.Equal(t, &SendTXArgs{
+		From: "0x0000000000000000000000000000000000000000",
+		Data: &hexutil.Bytes{90, 238, 148, 49},
```

Comment:
I think you should be able to change this to `&ethbinding.HexBytes` and have it work, which will remove the need to pull in `github.com/ethereum/go-ethereum`